server/utils/nunjucksSetup.ts

```diff
@@ -79,5 +81,109 @@ export default function nunjucksSetup(app: express.Application): nunjucks.Enviro
       checked: item === value,
     }))
   })
+
+  njkEnv.addFilter('toCourtTypeFilterCheckboxes', (courtTypes: CourtType[], allCourtsFilter: AllCourtsFilter) => {
```

Comment:
Generates GDS style checkboxes for the AllCourtsFilter, based on the spec. here: https://design-system.service.gov.uk/components/checkboxes/